src/service/db/schema.rs

```diff
@@ -27,6 +27,8 @@ fn mk_key(org_id: &str, stream_type: StreamType, stream_name: &str) -> String {
     format!("/schema/{org_id}/{stream_type}/{stream_name}")
 }
 
+// TODO: change the type of the returned value to `Result<Option<Schema>>,
```

Comment:
You can't return an `Option`, because the code need a empty schema when it not exists.